backend/app/exporters/serializers/ead3.rb

```diff
@@ -1345,56 +1349,76 @@ def serialize_container(inst, xml, fragments)
-
-    atts = {}
-
-    content = ""
-    content << title if title
-    content << ": " if date['expression'] || date['begin']
-    if date['expression']
-      content << date['expression']
-    elsif date['begin']
-      content << date['begin']
-      if date['end'] != date['begin']
-        content << "-#{date['end']}"
-      end
-    end
-
-    atts['linktitle'] = digital_object['title'] if digital_object['title']
-
-    if digital_object['digital_object_type']
-      atts['daotype'] = 'otherdaotype'
-      atts['otherdaotype'] = digital_object['digital_object_type']
-    else
-      atts['daotype'] = 'unknown'
-    end
-
-    if file_versions.empty?
-      atts['href'] = digital_object['digital_object_id']
-      atts['actuate'] = 'onrequest'
-      atts['show'] = 'new'
-      atts['audience'] = 'internal' unless is_digital_object_published?(digital_object)
-      xml.dao(atts) {
-        xml.descriptivenote { sanitize_mixed_content(content, xml, fragments, true) } if content
-      }
-    else
-      file_versions.each do |file_version|
-        atts['href'] = file_version['file_uri'] || digital_object['digital_object_id']
-        atts['actuate'] = (file_version['xlink_actuate_attribute'].respond_to?(:downcase) && file_version['xlink_actuate_attribute'].downcase) || 'onrequest'
-        atts['show'] = (file_version['xlink_show_attribute'].respond_to?(:downcase) && file_version['xlink_show_attribute'].downcase) || 'new'
-        atts['localtype'] = file_version['use_statement'] if file_version['use_statement']
-        atts['audience'] = 'internal' unless is_digital_object_published?(digital_object, file_version)
-        xml.dao(atts) {
-          xml.descriptivenote { sanitize_mixed_content(content, xml, fragments, true) } if content
-        }
-      end
-    end
+   return if digital_object["publish"] === false && !@include_unpublished
+   return if digital_object["suppressed"] === true
+
+   # ANW-285: Only serialize file versions that are published, unless include_unpublished flag is set
+   # already added to EAD2002 exporter, but not previously added to EAD3 exporter.
+   file_versions_to_display = digital_object['file_versions'].select {|fv| fv['publish'] == true || @include_unpublished }
+
+   title = digital_object['title']
+   date = digital_object['dates'][0] || {}
+
+   atts = {}
+
+   content = ""
+   content << title if title
+   content << ": " if date['expression'] || date['begin']
+   if date['expression']
+     content << date['expression']
+   elsif date['begin']
+     content << date['begin']
+     if date['end'] != date['begin']
+       content << "-#{date['end']}"
+     end
+   end
+
+   #the title is already added to descriptivenote/p, so no need to repeat here. let's use the new identifier attribute in EAD3
+   atts['identifier'] = digital_object['digital_object_id'] if digital_object['digital_object_id']
+   # and let's keep those URIs in the same place everywhere...
+   atts['altrender'] = digital_object['uri']
```

Comment:
Also, I could remove the altrender attributes, if that's preferred.  Just let me know.  In our local exports, I use altrender not only for digital objects, but also for the resource, archival object, and top container ASpace URIs.  Only the digital object URIs have been added to the current pull request, though.